types/stats.js/index.d.ts

```diff
@@ -29,6 +29,6 @@ declare namespace Stats {
     }
 }
 
-declare module 'stats.js' {
+declare module 'stats-js' {
```

Comment:
The folder should be (at least) renamed to `stats-js`.

`stats-js` is also now exported as a UMD bundle so this definition should be refactored like a commonjs definition (`export =` but at the top level, without `declare module`s) with an additional `export as namespace Stats`, but that could be done separately.

Reply:
@Jessidhia 
rename folder and so on. Could you re-review?